src/content/en/updates/2019/03/kv-storage.md

```diff
@@ -289,9 +299,11 @@ features mentioned in this article:
 - [Import Maps](https://github.com/WICG/import-maps)
 
 If your site currently uses `localStorage`, you should try switching to the KV
-Storage API, and if you sign up for the [KV Storage origin
+Storage API to see if it meets all your needs. And if you sign up for the [KV
+Storage origin
 trial](https://developers.chrome.com/origintrials/#/trials/active), you can
-actually deploy your changes today! All your users should benefit from better
-performance, and Chrome 74+ users won't have to pay any extra download cost.
+actually deploy these features today. All your users should benefit from better
```

Comment:
Not sure I understand this change; just reduces the enthusiasm.

Reply:
The main change is the addition of "to see if it meets all your needs". IIRC I was thinking we should add that because we know there may be use cases the KV Storage API can't handle (e.g. when the page is unloading).

Is there another wording you'd prefer?

Reply:
The chunk above makes sense, it's just the "your changes" -> "these features" and period -> exclamation point that I was confused about. But, this is minor; not worth doing a revet commit.